packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -36,6 +36,7 @@ export class PostgresIndexApi implements DatabaseIndexApi {
       .insert({
         stream_id: String(args.streamID),
         controller_did: String(args.controller),
+        stream_content: JSON.stringify(args.streamContent),
```

Comment:
Unsure about this; if we just want to use `String` instead since from what've seen stream content is already coming in as a string and Postgres does the validation itself on the field too.

Reply:
sorry, I don't really understand what the two options are?

Reply:
It's either to have cast a straightforward `String(args.streamContent)` or the current active conversion via `JSON.stringify(args.streamContent)`.

Reply:
this is resolved with the object comment.